### PR TITLE
[SPARK] Test parquet table creation by Spark 3.5.3

### PR DESCRIPTION
**Pull request by amaliujia**
This PR is to test parquet table creation by create table or create table select as

### CODE REVIEW

File: connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/UCSingleCatalog.scala [hunk 1]
```diff
@@ -226,7 +226,7 @@ private class UCProxy extends TableCatalog with SupportsNamespaces {
     val format: String = properties.get("provider")
     createTable.setDataSourceFormat(convertDatasourceFormat(format))
     tablesApi.createTable(createTable)
-    null
+    loadTable(ident)
```
Comment (amaliujia) [inline connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/UCSingleCatalog.scala:229]: Will revert this

File: build.sbt [hunk 2]
```diff
@@ -44,6 +44,7 @@ lazy val commonSettings = Seq(
     "org.apache.logging.log4j" % "log4j-api" % "2.23.1"
   ),
   resolvers += Resolver.mavenLocal,
+  resolvers += "Apache Spark 3.5.3 (RC2) Staging" at "https://repository.apache.org/content/repositories/orgapachespark-1466/",
```
Comment (cloud-fan) [inline build.sbt:47]: let's also test Delta 3.2.1 RC1: `resolvers += "Delta 3.2.1 (RC1) Staging" at https://oss.sonatype.org/content/repositories/iodelta-1166`

Comment (cloud-fan) [reply build.sbt:47]: there are some ignored tests in `TableReadWriteTest`, we can enable them now with this RC.

Comment (amaliujia) [reply build.sbt:47]: sure

File: connectors/spark/src/test/java/io/unitycatalog/connectors/spark/TableReadWriteTest.java [hunk 3]
```diff
@@ -154,7 +153,7 @@ public void testCredentialDelta(String scheme) throws ApiException, IOException
     session.stop();
   }
 
-  @Disabled("Ignoring test until Delta 3.2.1 is released.")
+//  @Disabled("Ignoring test until Delta 3.2.1 is released.")
```
Comment (cloud-fan) [inline connectors/spark/src/test/java/io/unitycatalog/connectors/spark/TableReadWriteTest.java]:
```suggestion
  @Disabled("Ignoring test until Delta 3.2.1 is released.")
```

**Review by cloud-fan — APPROVED** [review]

**Pull request merged**